docs/tutorials/grpo_with_pathways.md

```diff
@@ -29,6 +29,50 @@ Furthermore, we use Pathways for [orchestration](https://cloud.google.com/ai-hyp
 Follow instructions in [Install MaxText](https://github.com/AI-Hypercomputer/maxtext/blob/main/docs/guides/install_maxtext.md), but 
 recommend creating the virtual environment outside the `maxtext` directory.
 
+
+## Setup the following environment variables before running GRPO
+
+Setup following environment variables before running GRPO
+
+```bash
+# -- Model configuration --
+export MODEL=<model name> # e.g., 'llama3.1-70b'
```

Comment:
could you please specify that `MODEL` here can be `llama3.1-8b-Instruct` version, but `MODEL` in the command below needs to be non Instruct version, for e.g, `llama3.1-8b`

This is important because for RL, most likely people would use intruction tuned models

Reply:
I am thinking we should redefine `MODEL` and define `TOKENIZER` right before the `train_rl.py` because `TOKENIZER ` is not used in the checkpoint conversion and `MODEL` needs to called without `-Instruct` there